L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -65,13 +65,12 @@ void L1TMuonEndCapTrackProducer::produce(edm::Event& ev,
   //fprintf (write,"12345\n"); //<-- part of printing text file to send verilog code, not needed if George's package is included
 
   //std::auto_ptr<L1TMuon::InternalTrackCollection> FoundTracks (new L1TMuon::InternalTrackCollection);
-  auto FoundTracks = std::make_unique<L1TMuon::InternalTrackCollection>();
-  auto OutputCands = std::make_unique<l1t::RegionalMuonCandBxCollection>();
-  auto OutTracks = std::make_unique<l1t::EMTFTrackCollection>();
-  auto OutHits = std::make_unique<l1t::EMTFHitCollection>();
-  auto OutputTracks = std::make_unique<l1t::EMTFTrackExtraCollection>();
-  auto OutputHits = std::make_unique<l1t::EMTFHitExtraCollection>();
-  auto OutputHitsRPC = std::make_unique<l1t::EMTFHitExtraCollection>();
+  std::auto_ptr<l1t::RegionalMuonCandBxCollection > OutputCands (new l1t::RegionalMuonCandBxCollection);
+  std::auto_ptr<l1t::EMTFTrackCollection> OutTracks (new l1t::EMTFTrackCollection);
+  std::auto_ptr<l1t::EMTFHitCollection> OutHits (new l1t::EMTFHitCollection);
+  std::auto_ptr<l1t::EMTFTrackExtraCollection> OutputTracks (new l1t::EMTFTrackExtraCollection);
+  std::auto_ptr<l1t::EMTFHitExtraCollection> OutputHits (new l1t::EMTFHitExtraCollection);
+  std::auto_ptr<l1t::EMTFHitExtraCollection> OutputHitsRPC (new l1t::EMTFHitExtraCollection);
 
```

Comment:
@EsmaeelEskandari this PR changes the EMTF emulator which should not happen in a DQM related PR. If you want to change something in the emulator you should do a PR to cms-l1t-offline first, and @rekovic or @mulhearn will handle the PR to CMSSW.
In this case it looks like you are trying to undo a commit of @rekovic https://github.com/cms-sw/cmssw/commit/04ea88bdc6efbdea8c1f1a49f18209b9c2ba90b9 from 2 months ago.
Please check more carefully what changes are in your branch before making a PR.